lib/matplotlib/pyplot.py

```diff
@@ -432,7 +432,7 @@ def figure(num=None,  # autoincrement if None, else integer from 1-N
         If num is a string, the window title will be set to this figure's
         `num`.
 
-    figsize : tuple of integers, optional, default: None
+    figsize : tuple of floats, optional, default: None
```

Comment:
```suggestion
    figsize : (float, float), optional, default: None
```

Reply:
Thanks for th PR. In more recent times, we‘ve decided to explicitly write out 2-tuples as it‘s more precise than just tuplebof floats.